CControl_Handler.cc

```diff
@@ -166,14 +200,21 @@ bsoncxx::document::value CControl_Handler::GetStatusDoc(std::string hostname){
              << "delay" << fDDC10->GetHEVOptions().delay
              << bsoncxx::builder::stream::close_document;
   }
-
-  // Here you would add the V1495...
+  // Write the settings for the Muon Veto V1495 board into status doc 
+  if(fV1495 != NULL){
+     in_array << bsoncxx::builder::stream::open_document
+	      << "type" << "V1495"
+	      << "Module reset" << fOptions->GetRegisters(fBID)[0].val
+	      << "Mask A" << fOptions->GetRegisters(fBID)[1].val
+	      << "Mask B" << fOptions->GetRegisters(fBID)[2].val
+	      << "Mask D" << fOptions->GetRegisters(fBID)[3].val
+	      << "Majority Threshold" << fOptions->GetRegisters(fBID)[4].val
+	      << "Coincidence Window" << fOptions->GetRegisters(fBID)[5].val
+	      << "NIM/TTL CTRL" << fOptions->GetRegisters(fBID)[6].val
```

Comment:
Rather than 7 calls to GetRegisters, might be better to call once and cache the return.